config.py

```diff
@@ -50,8 +50,10 @@ class Config(object):
     ERROR_404_HELP = False
     CSRF_ENABLED = True
     SERVER_NAME = env('SERVER_NAME', default=None)
+    STATIC_DOMAIN = env('STATIC_DOMAIN', default=SERVER_NAME)
```

Comment:
And this should be part of the app settings. Not config. Modifiable at will by the admin.